migration/incremental/015-tracing-redesign.sql

```diff
@@ -304,32 +304,27 @@ $do$
 DECLARE
     _pg_version int4 := current_setting('server_version_num')::int4;
 BEGIN
+    SET search_path = pg_catalog;
```

Comment:
But this shouldn't be needed due to https://github.com/timescale/promscale_extension/blob/develop/migration/bootstrap/000-migration-table.sql#L2
```suggestion
    SET search_path = pg_catalog, pg_temp;
```